web3sTests/TestConfig.swift

```diff
@@ -13,11 +13,11 @@ struct TestConfig {
     // This is the proxy wss URL for connecting to the Blockchain. For testing we usually use the Goerli network on Infura. Using free tier, so might hit rate limits
     static let wssUrl = "wss://goerli.infura.io/ws/v3/b2f4b3f635d8425c96854c3d28ba6bb0"
 
-    // An EOA with some Ether, so that we can test sending transactions (pay for gas)
-    static let privateKey = "0xef4e182ae2cf32192d2a62c1159c8c4f7f2d658c303d0dfca5791a205456a132"
+    // An EOA with some Ether, so that we can test sending transactions (pay for gas). Set by CI
+//    static let privateKey = "SET_YOUR_KEY_HERE"
 
     // This is the expected public key (address) from the above private key
-    static let publicKey = "0x719561fee351F7aC6560D0302aE415FfBEEc0B51"
+    static let publicKey = "0xE78e5ecb061fE3DD1672dDDA7b5116213B23B99A"
```

Comment:
Ok, so this will have to change to "match" the private key used, right?

Reply:
Yes, that one too.